packages/components/inputs/selectable-search-input/src/selectable-search-input.story.js

```diff
@@ -112,15 +108,6 @@ storiesOf('Components|Inputs', module)
             id={text('id', '')}
             name={name}
             value={value}
-            onChange={(event) => {
```

Comment:
**nit**: we might want to keep the call to the `action` as we had before so we can check the callback is indeed called from the Storybook interface.